apps/manager_app/manager_app.c

```diff
@@ -116,6 +118,7 @@ void manager_app_main(usb_event_t usb_evt) {
       break;
     }
     case MANAGER_QUERY_AUTH_CARD_TAG: {
+      card_auth_handler(&usb_evt);
```

Comment:
Hi Aman, as we have already decoded the query, I think we should pass reference to that. It will also make it consistent with device_info_flow and device_authentication flow.

Reply:
I was working on a change that would require this, will push tomorrow once tested

Reply:
I was working on a change that would require this, will push in the morning once tested

Reply:
updated